ui/src/app/applications/components/application-sync-panel/application-sync-panel.tsx

```diff
@@ -172,9 +172,27 @@ export const ApplicationSyncPanel = ({application, selectedResource, hide}: {app
                                                 .filter(item => !item.hook)
                                                 .map((item, i) => {
                                                     const resKey = nodeKey(item);
+                                                    const contentStart = resKey.substr(0, Math.floor(resKey.length / 2));
+                                                    let contentEnd = resKey.substr(-Math.floor(resKey.length / 2));
+                                                    // Fix Latin letters being rendered left to right on ellipsis with direction set to rtl
```

Comment:
@ciiay reached out to me and explained that this is tied to the css change.

A comment in both spots explaining that it's relying on the direction change to place the ellipsis where we want it would help.

And then this comment after it can explain "and because we've direction swapped, we now trip on strong LTR characters, so we need to do a bit of extra work for them".